packages/aws-cdk-lib/aws-ecs/lib/container-definition.ts

```diff
@@ -538,6 +553,14 @@ export class ContainerDefinition extends Construct {
       }
     }
 
+    if (props.credentialSpecs) {
```

Comment:
Based off of a previous comment, we should have a synth-time check (+ unit test) for ensuring that `credentialSpecs.length === 1`. We never want to silently ignore additional info

Reply:
I wanted to avoid the need to create a new PR if a new feature starts using more than one CredSpec in the future. But if you really insist, I made the change and added the appropriate unit test.